core/store/presenters/presenters.go

```diff
@@ -495,20 +495,20 @@ type Tx struct {
 }
 
 // NewTx builds a transaction presenter.
-func NewTx(tx *models.Tx) Tx {
+func NewTx(tx *models.Tx) (Tx, error) {
```

Comment:
Optional: This no longer needs to return a tuple with an error.

Reply:
Going to make a small follow up and will add this.